openapi/resource_info.go

```diff
@@ -325,6 +328,32 @@ func (r resourceInfo) getPollingStatuses(response spec.Response, extension strin
 	return statuses, nil
 }
 
+func (r resourceInfo) getResourceTimeout(operation *spec.Operation) (*time.Duration, error) {
+	if operation == nil {
+		return nil, nil
+	}
+	return r.getTimeDuration(operation.Extensions, extTfResourceTimeout)
+}
+
+func (r resourceInfo) getTimeDuration(extensions spec.Extensions, extension string) (*time.Duration, error) {
+	if value, exists := extensions.GetString(extension); exists {
+		regex, err := regexp.Compile("^[\\d]+([\\.]{1}[\\d]+)?[smh]{1}$")
```

Comment:
Could this be simplified to `^\d+(\.\d+)?[smh]{1}$`?

Reply:
awesome, updated